src/world/message_handler.h

```diff
@@ -23,6 +23,8 @@ along with this program.  If not, see http://www.gnu.org/licenses/
 
 #include "common/mmo.h"
 
+#include <netinet/in.h>
```

Comment:
Is this valid for Win, OSX and Linux? This looks suspiciously like it should live with mmo or socket

Reply:
I was getting a compile issue on mac os, but only after this change for some reason. Ill double check

Reply:
Used ifdef win32 as in other files, with winsock